packages/schemas/resources/css/components/fused-group.css

```
@@ -6,6 +6,18 @@
     & .fi-sc {
         @apply gap-px rounded-lg bg-gray-950/10 dark:bg-white/20;
 
+        .fi-sc-component {
+            @apply h-full;
+
+            & > .fi-fo-field {
+                @apply h-full;
+            }
```

I think this works and is more consistent, right?

```suggestion
        & .fi-sc-component {
            @apply h-full;

            & .fi-fo-field {
                @apply h-full;
            }
```

---

Am not so at home in this type of CSS, but appears to work good as well!